composer.json

```diff
@@ -12,10 +12,8 @@
         }
     ],
     "require": {
-        "phpspec/prophecy": "~1.5"
-    },
-    "require-dev": {
-        "atoum/atoum": "~2.0"
+        "phpspec/prophecy": "~1.5",
+        "atoum/atoum": "^2.0"
```

Comment:
@Hywan nope: in order to make composer check dependencies, extensiosn have to define atoum as a required dependency. Composer resolve dependency tree by merging the root package required and dev. required dependencies and children's required deps.